app/src/main/res/layout/save_fragment.xml

```diff
@@ -106,13 +106,26 @@
                         android:layout_height="match_parent"
                         android:text="@string/unlock_with_lock_screen" />
 
+                    <RadioButton
+                        android:id="@+id/unlock_device_credentials"
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:text="@string/unlock_with_device_credentials" />
+
                     <RadioButton
                         android:id="@+id/unlock_biometrics"
                         android:layout_width="match_parent"
                         android:layout_height="match_parent"
                         android:enabled="@{canUseBiometrics}"
                         android:text="@string/unlock_with_biometrics" />
 
+                    <TextView
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:textAppearance="@style/TextAppearance.DiceKeys.Caption"
+                        android:layout_marginStart="30dp"
+                        android:text="*Encryption key will be invalidated on new biometric enrollment."/>
```

Comment:
The encryption key is not what the user sees.  They just know that they lose their DiceKey if they add new biometrics. 
 Is it accurate to say:

> Stored DiceKeys will be erased if you add new fingerprints, faceprints, or other biometrics.